vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,195 @@
+---
+title: "Protein Structure Analysis Workflow"
+author: "Jan-Philipp Quast"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Protein Structure Analysis Workflow}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+<style>
+body {
+text-align: justify}
+</style>
+
+```{r, include = FALSE}
+test_protti <- identical(Sys.getenv("TEST_PROTTI"), "true")
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This vignette will take you through all important tools **protti** provides for structural data analysis. These include functions that allow you to fetch information about structures both on a general and on the atom level. Additionally, distances between atoms can be extracted from structures. Lastly, B-factors* of structures can be manipulated to contain custom values, which gives you the ability to colour regions on structures as well as apply different values associated with each amino acid as a colour gradient. 
```

Comment:
Here I would mention that the coloring has to be done with additional biomolecule visualization software